docs/src/contributing/documentation.md

```diff
@@ -1,21 +0,0 @@
-# Documentation
```

Comment:
We should probably add something like this, right? Or maybe link to the docs on how to use the apollo docs